python/tk_framework_desktopserver/sgtk_file_dialog.py

```diff
@@ -10,10 +10,7 @@
 
 import sys
 
-try:
-    from PySide import QtCore, QtGui
-except ImportError:
-    from PyQt4 import QtCore, QtGui
+from sgtk.platform.qt import QtCore, QtGui
```

Comment:
10 points for Gryffindor!

Reply:
I knew what you were quoting, but I was wondering if there was an [extra meaning](http://www.urbandictionary.com/define.php?term=10%20points%20for%20Gryffindor) in american culture I wasn't aware of.

Gotta love Urban Dictionary.